src/Controller/ArgumentResolver/AddressValueResolver.php

```diff
@@ -46,27 +47,35 @@ public function resolve(Request $request, ArgumentMetadata $argument)
 
         $user = $this->security->getUser();
 
-        if (!is_object($user)) {
-            // e.g. anonymous authentication
-            return yield null;
-        }
+        $data = base64_decode($value);
+        $data = get_object_vars(json_decode($data));
```

Comment:
I think `get_object_vars(json_decode($data))` can be simplified to `json_decode($data, true)` ([2nd parameter ](https://www.php.net/manual/en/function.json-decode.php)allows to return an associative array)

Reply:
Great, thanks